src/controls/controller.h

```diff
@@ -1,7 +1,27 @@
 #ifndef _CONTROLLER_H
 #define _CONTROLLER_H
 
-#include <ultra64.h>
+#include <stdint.h>
+
+// The button values match those defined by libultra - see os_cont.h. Instead of
+// hard-coding these values here, a better approach would be to have a dedicated
+// file for each library implementation and choose the appropriate one at build
+// time.
+// NOTE: libdragon uses the same values but reversed - see joypad_buttons_t.
+#define BUTTON_A       0x8000
```

Comment:
For clarity when working with these constants, please create an enum instead of using `#define`s.

Reply:
Done in 26b374e, thanks.